iamclient/iamclient.go

```diff
@@ -47,8 +47,9 @@ import (
  **********************************************************************************************************************/
 
 const (
-	iamRequestTimeout    = 30 * time.Second
-	iamReconnectInterval = 10 * time.Second
+	iamRequestTimeout      = 30 * time.Second
+	iamReconnectInterval   = 10 * time.Second
+	disconnectedFromIAMLog = "Disconnected from IAM"
```

Comment:
No need to put logs as const. Either suppress linter or better to rephrase in different use cases.